domain/relation/state/relation_test.go

```diff
@@ -346,6 +346,235 @@ func (s *relationSuite) TestGetRelationEndpointsRelationNotFound(c *gc.C) {
 	c.Assert(err, jc.ErrorIs, relationerrors.RelationNotFound)
 }
 
+func (s *relationSuite) TestGetRegularRelationUUIDByEndpointIdentifiers(c *gc.C) {
+	// Arrange: Add two endpoints and a relation on them.
+	expectedRelationUUID := "fake-relation-uuid"
+
+	charmRelationUUID1 := "fake-charm-relation-uuid-1"
+	applicationEndpointUUID1 := "fake-application-endpoint-uuid-1"
+	relationEndpointUUID1 := "fake-relation-endpoint-uuid-1"
```

Comment:
Use GenRelationUUID(c), GenApplicationID(c) and GenCharmID(c).

Reply:
See earlier comment.